showcase/app/styles/showcase-pages/code-block.scss

```
@@ -13,4 +13,8 @@ body.page-components-code-block {
       left: 0;
     }
   }
+
+    .shw-component-code-block-visibility-hidden {
```

```suggestion
  .shw-component-code-block-visibility-hidden {
```
[Nit] Indentation fix